tokio-postgres/src/connect_tls.rs

```diff
@@ -25,18 +26,20 @@ where
         SslMode::Prefer | SslMode::Require => {}
```

Comment:
Should there be any special behavior for `sslmode=prefer sslnegotiation=direct`?

Reply:
No. When sslmode is prefer or required, we go directly to ssl handshake just like other ssl based protocol, for example, https. And those TLS libraries will initialize the handshake process, we don't need to do anything here.